app/javascript/components/mappers/soqlToDomain/application.js

```diff
@@ -7,7 +7,7 @@ import { mapListing } from './listing'
 import { mapUser } from './user'
 import { mapShape, mapList } from '../utils'
 
-const parseList = text => split(text, ';')
+const parseList = text => { return (text ? split(text, ';') : []) }
```

Comment:
there was actually a bug where ADA info wasn't showing and saving correctly on supplemental applications